libs/utils/src/env.ts

```diff
@@ -0,0 +1,9 @@
+export const env = (envName: string) => {
+  const value = process.env[envName];
+  if (!value) {
+    throw new Error(
+      `Missing environment variable ${envName}, add it to your .env`
+    );
+  }
+  return value;
+};
```

Comment:
This makes sense as a framework level addition -- its _certain_ to be used in other apps.